spec/support/request/web_helper.rb

```diff
@@ -128,9 +128,11 @@ def close_select2
   end
 
   def tomselect_search_and_select(value, options)
-    page.find("[name='#{options[:from]}']").sibling(".ts-wrapper").click
-    page.find(:css, '.ts-dropdown input.dropdown-input').set(value)
-    page.find(:css, '.ts-dropdown .ts-dropdown-content .option', text: value).click
+    tomselect_wrapper = page.find("[name='#{options[:from]}']").sibling(".ts-wrapper")
```

Comment:
I think we have to do this instead of full page, since a page could contain multiple tom select elements and we might end up targeting wrong element.

Reply:
Yep. I think you're right.